public/javascripts/model.js

```diff
@@ -0,0 +1,33 @@
+var User = function(user){
+  this.name=user.name;
+  this.description = user.description;
+};
+
+var getAjax = function(event){
```

Comment:
Kind of confused about the name of this function - it returns a url only.  Good though.